openapidemo/app/src/main/java/com/intelligent/openapidemo/fragment/AlertListFragment.kt

```diff
@@ -62,7 +60,10 @@ class AlertListFragment : Fragment() ,AlertListAdapter.ItemClick {
 
         val alertListScreenViewModel =
             ViewModelProvider(this).get(AlertListViewModel::class.java)
-        activity?.let { alertListScreenViewModel.getAlertList(it, buildingId) }
+        activity?.let { context?.let { it1 ->
```

Comment:
Activity context is sufficient. No need to extract context separately. Why there are 2 contexts passed ? One is sufficient.